config/service.py

```diff
@@ -286,6 +286,18 @@
 
 PRIMARY_NAVIGATION = [
     {
+        "label": "About",
+        "url": {
+            "url": "https://monitor.jisc.ac.uk/uk/about",
+        },
+        "visibility": {
+            "auth": True,
+            "anonymous": False
```

Comment:
Shouldn't these tabs show all the time and not require auth? This choice seems wrong.